installed-versions.json

```diff
@@ -1,14 +1,20 @@
 {
+  "version": "6.0.0",
   "name": "idp-profile-ui",
   "dependencies": {
     "@babel/polyfill": {
       "version": "7.12.1",
       "resolved": "https://registry.npmjs.org/@babel/polyfill/-/polyfill-7.12.1.tgz",
       "overridden": false
     },
+    "@sentry/vue": {
```

Comment:
I don't understand why vue is just now showing up as a dependency.

Reply:
I forgot to run `make depsupdate` after installing sentry and probably other times dependencies were changed. Probably another thing to add to a checklist in a PR template.